src/options/orders.ts

```diff
@@ -10,7 +9,7 @@ export interface OrderCountOptions extends DateOptions {
 
     financial_status?: FinancialStatus;
 
-    fulfillment_status?: FulfillmentStatus;
+    fulfillment_status?: "shipped" | "partial" | "unshipped" | "any" | "unfulfilled";
```

Comment:
Can you add a note here that the fulfillment_status is different so that we do not forget this?